packages/manager/modules/logs-to-customer/src/components/data-streams/DataStreamAccountLink.component.tsx

```diff
@@ -0,0 +1,29 @@
+import { Links, LinkType } from '@ovh-ux/manager-react-components';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import { OdsHTMLAnchorElementTarget } from '@ovhcloud/ods-common-core';
+import React, { useContext, useEffect, useState } from 'react';
+import { Service } from '../../data/types/dbaas/logs';
+
+const DataStreamAccountLink = ({ service }: { service: Service }) => {
+  const [accountURL, setAccountURL] = useState('');
+  const {
+    shell: { navigation },
+  } = useContext(ShellContext);
+
+  useEffect(() => {
+    navigation
+      .getURL('dedicated', `#/dbaas/logs/${service.serviceName}`, {})
+      .then(setAccountURL);
+  }, [navigation]);
```

Comment:
If you want you can now use : 

https://github.com/ovh/manager/pull/14964

```suggestion
  const {data: accountUrl} = useGetUrlNavigation('dedicated', `#/dbaas/logs/${service.serviceName}`, {});
```

Reply:
Good idea, but we'll have to wait ods18 migration to be able to use the latest core features.